pom.xml

```diff
@@ -1209,9 +1209,6 @@ under the License.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-project-info-reports-plugin</artifactId>
-        <configuration>
-          <dependencyLocationsEnabled>false</dependencyLocationsEnabled><!-- waiting for MPIR-267 -->
-        </configuration>
```

Comment:
The issue has been autoclosed, Is this the right thing to do?

Reply:
Yes was removed in https://issues.apache.org/jira/browse/MPIR-367
Wrong comments

Reply:
not wrong comment: the real issue is MPIR-267
but I discover that the problem was "solved" by removing the feature in MPIR-367

then yes, no need to keep the configuration, thanks for noticing this